config/routes.rb

```diff
@@ -8,4 +9,7 @@
   get 'categories', to: 'fee_categories#index', as: 'categories'
   get 'categories/:id', to: 'fee_categories#show', as: 'category'
   root to: 'fee_categories#index'
+
+  get 'search/fees'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.